tui/tests/test_weather.py

```
@@ -0,0 +1,488 @@
+        async with app.run_test() as pilot:
+            assert app.title == "Hookwise"
+            bg = app.query_one("#weather-bg", WeatherBackground)
+            assert bg is not None
+
+    async def test_weather_cycle_key(self):
+        """'w' key cycles through weather conditions."""
+        from hookwise_tui.app import HookwiseTUI
+        app = HookwiseTUI()
+        async with app.run_test() as pilot:
+            bg = app.query_one("#weather-bg", WeatherBackground)
+            initial_weather = bg.weather
```

Ruff (0.15.2)

[error] Local variable `initial_weather` is assigned to but never used

Remove assignment to unused variable `initial_weather`

(F841)